### PR TITLE
Feat/roadmap

### PR DESCRIPTION
**Pull request by MatiasOS**
### Components
- Add Roadmap to Foundation page

There are some issues that I can't fix

- [x] `mdi-circle-slice-4` is not showing. I tried to change the Icon (with mdi-bitocoin) and works properly. You can look at

- [x] In current homepage, there is an `::after` that add some dots between period section. I haven't add it. Don't know how to do it properly. Suggestion are welcome.

- [x] Color of `text-muted` look different

- [x] Padding in `Platform` group not the same as the original version, please take a look.

### CODE REVIEW

File: src/features/Foundation/Roadmap/Group.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const Group = (props: GroupType) => {
+  const {
+    title, elements, main, muted,
+  } = props;
+  const isMain = main ? 'main-group h3 pt-1' : 'pt-3';
+  const isMuted = muted ? 'text-muted' : '';
+  const isPadded = main ? 'pl-5' : '';
+  return (
+    <div className>
+      <h4 className={`text-center ${isMuted} ${isMain}`}>{title}</h4>
```
Comment (PabloAlice) [inline src/features/Foundation/Roadmap/Group.jsx]: you can use classnames package for these cases

File: src/features/Foundation/Roadmap/Item.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const Item = (props: ItemType) => {
+  const {
+    done, text, buttonLabel, href, muted, isMainButton,
+  } = props;
+  const icon = done ? 'mdi-checkbox-marked-circle' : 'mdi-circle-slice-4';
+  const isMuted = muted ? 'text-muted' : 'roadmap-item';
+  const buttonVariant = isMainButton ? 'primary' : 'dark';
+  return (
+    <>
+      <i className={`mdi ${icon} ${isMuted}`} />
```
Comment (PabloAlice) [inline src/features/Foundation/Roadmap/Item.jsx]: same here

File: src/features/Foundation/Roadmap/Period.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  constructor(props: PeriodType) {
+    super(props);
+    this.state = {
+      isOpened: props.isOpened,
+    };
+  }
+
+  handleCollapseClick = () => {
+    const { isOpened } = this.state;
+    this.setState({ isOpened: !isOpened });
```
Comment (PabloAlice) [inline src/features/Foundation/Roadmap/Period.jsx]: check this, you should use the callback form.
```javascript
this.setState(prevState => ({ isOpened: !prevState.isOpened }));
```
https://reactjs.org/docs/react-component.html#setstate

**Comment by menschikov** [comment]
@skkr please fix styles

**Comment by menschikov** [comment]
@skkr please review it and help with styles
@MatiasOS I pushed some fixes for paddings, please check mobile view, the Platform section should be without left padding on -sm (all section should be centered with text aligned to left (see current website). Also Mati, clear state on page refresh, page remembers expanded section

File: src/features/Foundation/Roadmap/Group.jsx (hunk 1 above)
Comment (MatiasOS) [reply src/features/Foundation/Roadmap/Group.jsx]: Done

File: src/features/Foundation/Roadmap/Period.jsx (hunk 3 above)
Comment (MatiasOS) [reply src/features/Foundation/Roadmap/Period.jsx]: Done

**Comment by menschikov** [comment]
@skkr please help with this part 
image

**Pull request merged**